docs/tutorial.rst

```diff
@@ -1224,65 +1224,54 @@ of at least 0.8 by the end. Next, we'll walk through the steps required to do th
     this conditioning works by re-running the simulation until the condition is
     achieved, a nearly impossible condition will result in very long run times.
 
-First, we need to define the mutation type for the selected mutation.
-So we can refer to it later, we need its "mutation type ID". This is just
-the index of the new mutation type in the contig's list of mutation types.
+First, we need to add a single site mutation in the middle of the contig.
+This represents a mutation of a single base pair, with associated fitness and
+dominance coefficients.  The single site mutation must have a unique string ID
+so that we can refer to it later.
```

Comment:
This doesn't actually add the mutation, it just adds, like, the 'potential mutation' or something. How about
```suggestion
To set things up, we need to first add the site at which the selected mutation
will occur. This is like adding a DFE, except to a single site - we're saying that
there's a potential mutation at the given site that has these fitness consequences.
So that we can refer to it later, we give it unique string ID. Here, we'll add the site
in the middle of the contig, with ID "hard sweep".
```